en/flight_controller/pixhawk4.md

```diff
@@ -37,58 +37,66 @@ Pixhawk<sup>&reg;</sup> 4 is an advanced autopilot designed and made in collabor
 * Other Characteristics:
   * Operating temperature: -40 ~ 85°c
 
-Download full Pixhawk 4 Technical Data Sheet from [here]().
-<!-- Later change to holybro wesbite (https://github.com/PX4/Hardware/blob/master/FMUv5/Pixhawk4-Datasheet.pdf).-->
-
-## Assembly/Setup 
-
-Pixhawk 4 Wiring Quick Start is provided [here](../assembly/quick_start_pixhawk4.md). It includes instructions on how to assemble important peripherals such as GPS, Power Management Board etc.
+Additional information can be found in the [Pixhawk 4 Technical Data Sheet]().
+<!-- Later change to holybro wesbite (https://github.com/PX4/Hardware/blob/master/FMUv5/Pixhawk4-Datasheet.pdf). -->
 
 
-## Build Instructions
+## Purchase
 
-`make px4fmu-v5_default upload`
+Order from [Holybro](https://shop.holybro.com/pixhawk-4beta-launch_p1089.html).
 
 
-## Purchase
-Order from [Holybro](https://shop.holybro.com/pixhawk-4beta-launch_p1089.html)
+## Connectors
 
+![Pixhawk 4 connectors](../../assets/flight_controller/pixhawk4/pixhawk4-connectors.jpg)
 
-## Connectors
-> **Note** The SBUS RC and PPM RC ports are for RC receivers only and provide power. NEVER connect any servos, power supplies or batteries to it 
-or to the receiver connected to it.
+> **Warning** The **DSM/SBUS RC** and **PPM RC** ports are for RC receivers only. These are powered! NEVER connect any servos, power supplies or batteries (or to any connected receiver).
 
-![Pixhawk 4 connectors](../../assets/hardware/hardware-pixhawk4-connectors.png)
 
 ## Pinouts
 
-Download Pixhawk 4 pinouts from [here]().
+Download *Pixhawk 4* pinouts from [here]( ).
```

Comment:
@barzanisar This link does not work, and I don't have have access to the github one either (so no one much will). Can we get this public?